fixtures/fos/lifting-ball-dmx.json

```
@@ -0,0 +1,159 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Lifting Ball DMX",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Frank"],
+    "createDate": "2020-06-08",
+    "lastModifyDate": "2020-06-08",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2020-06-08",
+      "comment": "created by Q Light Controller Plus (version 4.12.2)"
+    }
+  },
+  "physical": {
+    "dimensions": [320, 343, 150],
+    "weight": 7.4,
+    "power": 120,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "None"
+    }
+  },
+  "matrix": {
+    "pixelCount": [
+      null,
+      null,
+      1
+    ]
+  },
+  "availableChannels": {
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0%",
+        "angleEnd": "100%",
+        "helpWanted": "Can you provide exact angles?"
```

```suggestion
        "angleEnd": "100%"
```

This is a winch, so it's not angles.  Is "Tilt" the correct type to use for this?